### PR TITLE
Add iOS walkthrough

### PR DESCRIPTION
**Pull request by karimnaaji**
Add a new page for iOS walkthrough.

### CODE REVIEW

**Review by matteblair — COMMENTED** [review]
I just got around to checking this out, it's great! This will be a big help to iOS users. 

The only change I'd suggest is to remove the API key used in the example code and just use something like `<YOUR_API_KEY_HERE>` instead.

**Comment by karimnaaji** [comment]
Yep, good call! Will update this.

File: pages/iOS-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+![](images/iOS-Simulator.png)
+
+### Adding Tangram to an iOS application in 5 steps
+
+1. **Add the tangram binary Framework to your XCode project**. 
+
+Using the recommended way of using the framework with CocoaPods, adding the Tangram Pod to your app is as simple as adding the following to your `Podfile`:
+
+```sh
+pod 'Tangram-es', '~> 0.6.0'
```
Comment (tallytalwar) [inline pages/iOS-walkthrough.md:33]: Maybe now update this to `0.6.1`.

**Review by tallytalwar — COMMENTED** [review]
I was able to mimic the entire thing. 👍 

Few things I had to figure out:
1. using xcode.xcworkspace project (but @karimnaaji mentioned users of cocoapod know this)
2. to use the `Main.storyboard` of the default project instead of `LaunchScreen.storyboard` to create the `GLKView`. We should clarify this if its already not a known ios world thingy.

**Pull request merged**